src/components/RichTextEditor/plugins/mark/markPlugin.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2025-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { markPlugin as baseMarkPlugin, marks, toggleMark } from "@ndla/editor";
+import { isKeyHotkey } from "is-hotkey";
+
+export const markPlugin = baseMarkPlugin.configure({
+  shortcuts: {
+    toggleBold: {
+      keyCondition: isKeyHotkey("mod+b"),
```

Comment:
Dette er einaste plassen der hotkey er definert. Hadde vore fint med en mouseover på knappen feks.

Reply:
Tenker å lage en utility-funksjon som lager tooltips for oss!